planning-release.md

```diff
@@ -0,0 +1,23 @@
+# Planning a release on XebiaLabs
```

Comment:
As user, I think there are should have a introduce in README to provide some high-level information about this Repo:

1.  Add an agenda in README which help user know which section they are going to read. 
2. Provide some brief information about "XebiaLabs-Appium-ReactNative", e.g What will this Repo show to the end user.